ph5/utilities/resp_load.py

```diff
@@ -138,11 +144,6 @@ def create_list(self):
                             bit_weight = Response_t['bit_weight/value_d']
                             bit_weight_units = Response_t['bit_weight/units_s']
                             gain_units = Response_t['gain/units_s']
-                        else:
-                            LOGGER.warning(
-                                "No Response table found for das {0} channel "
-                                "{1}.\n".format(serial, channel))
-                        try:
```

Comment:
Please keep the warning.  It assists with troubleshooting.

Reply:
If you run the `resp_load` program on a master.ph5 file that hasn't had time series data loaded yet, you'll potentially receiver thousands of lines of warnings. IMO, this just becomes noise. The `ph5_validate` program will tell you if there is a das without data, but I don't think this is the job of `resp_load`.

Is there a reason that data needs to be loaded before a RESP file is added to a master.ph5? Does it matter if someone loads metadata into a master.ph5 before loading the time series?

Reply:
The PIC-distributed procedure tells users that data loading is one of the very first steps.  So it is current SOP to expect that data will be there when we run resp_load on a normal experiment.

Sure, ph5_validate can catch it, but by that time I've got more kefs I have to go back and check and probably delete and generate again.  It is easier to catch it at resp_load, and ph5_validate has enough buggy warnings (like the 'no data for this channel') that a warning about a das having no response table could become lost in the noise.

The question is now what harm is caused by keeping the warning as it is in resp_load until other bugs (like response_table_n_i indexing and the ph5_validate one I mentioned above) are smoothed out.  I would suggest that it causes no harm and doesn't impact the usefulness of resp_load for an archive being built according to current SOP.

Reply:
Thanks Holly. Would you mind if I just add a flag where a user can optionally silent warnings?